xml/security_kerberos.xml

```diff
@@ -1914,6 +1914,140 @@ Valid starting     Expires            Service principal
+    </step>
+    <step>
+     <para>
+Execute the action that triggers the &krb; authentication issue for example, logging into a service.
+The trace log captures the process details.
+     </para>
+    </step>
+    <step>
+        <para>
+   Examine in-depth, the contents of the log file as specified by <envar>KRB5_TRACE</envar>. The log file contains details
+   about &krb; requests,responses and errors. Specifically, look for:
+        </para>
+        <variablelist>
+            <varlistentry>
+             <term>Request and Response Details </term>
+             <listitem>
+              <para>
+               Check if requests are reaching the KDC and responses are being received.
+              </para>
+                   </listitem>
+            </varlistentry>
+            <varlistentry>
+             <term>Error messages</term>
+             <listitem>
+              <para>
+               Evaluate any error messages or codes that indicate something is wrong.
+              </para>
+
+             </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term>Network issues</term>
+                <listitem>
+                 <para>
+                  Check if the KDC is reachable or any network issues.
+                 </para>
+                </listitem>
+               </varlistentry>
+                </variablelist>
+       </step>
+       <step>
+        <para>
+   Examine the common issues such as:
+        </para>
+        <variablelist>
+            <varlistentry>
+             <term>Configuration issues </term>
+             <listitem>
+              <para>
+               Verify that the realms, KDC addresses and other settings are correct in the configuration file.
+              </para>
+                   </listitem>
+            </varlistentry>
+            <varlistentry>
+             <term>Synchronized time</term>
+             <listitem>
+              <para>
+               &krb; is sensitive to time discrepancies so ensure all systems have a synchronized clock.
+              </para>
+
+             </listitem>
+            </varlistentry>
+            <varlistentry>
+                <term>Network issues</term>
+                <listitem>
+                 <para>
+                 Verify that there is stable connection between the client and the KDC.
+                 </para>
+                </listitem>
+               </varlistentry>
+               <varlistentry>
+                <term>DNS issues</term>
+                <listitem>
+                 <para>
+                 Ensure there is a proper DNS resolution for the KDC and other &krb; related services.
+                 </para>
+                </listitem>
+               </varlistentry>
+               <varlistentry>
+                <term>Permission issues</term>
+                <listitem>
+                 <para>
+                Check if the &krb; principal has the right permissions.
```

Comment:
krb principals don't have permissions, what does this mean? KRB is authentication, not authorisation.